### ISSUE TITLE
Add Cake.Issues.Recipe

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.VsCode.Recipe#12]
We could add Cake.Issues.Recipe for handling of issues reported by ESLint. Requires https://github.com/cake-contrib/Cake.Issues.Recipe/issues/3 to be implemented first.

**Comment by gep13** [comment]
@pascalberger sounds like a plan to me.

What format is currently being used by the ESLint Addin?  Can't we just use that?

**Comment by pascalberger** [comment]
It currently supports [JSON formatter](https://cakeissues.net/docs/issue-providers/eslint/features)

**Comment by gep13** [comment]
Oh right, I see (I think)...

You are saying that the ESLint addin supports JOSN log files, but currently there isn't a way within the Cake.Issues.Recipe to bring those in.  Is that correct?

**Comment by pascalberger** [comment]
Yes, but it's not a big deal to add ESLint support to Cake.Issues.Recipe, that's quickly done. I think the biggest task to solve here is how to call the linting in a way that the recipe script knows the output path afterwards.

**Comment by gep13** [comment]
@pascalberger I know you like a challenge 😛

**Comment by pascalberger** [comment]
Cake.Issues.Recipe 0.4.4 has been released with support for ESLint json formatter output

**Comment by gep13** [comment]
@pascalberger woot!  Are you in a position to add this into the Recipe here?
